### DISCUSSION TITLE
midnight commander like copying etc betwen two panels available?

### DISCUSSION THREAD

**Discussion by jmlucjav** [sxyazi/yazi#440] [1. Q&A]
Hi,

I see this has already been requested: having twin panels side by side like mc:

> be able to make two tabs side-by-side (two tabs both only show the middle column)
> 
> This will be a challenge for image previews, as displaying 2 or more images on the same interface can be tricky.
> 
> The simplest way is to use the built-in split-screen feature of the terminal or terminal multiplexers like tmux and zellij (Yazi now supports displaying images within tmux and zellij).

The suggestion, IIUC, is to run two yazi processes in two shells, with a split terminal, so you get the same visual effect (and disabling some columns etc). 

My question is, if doing so, is it possible to for instance copy one file from left panel to right one and such? I guess some IPC would be needed between both yazi processes?

Thanks!

**Comment by sxyazi** [comment]
There's also a [related discussion on Discord](https://discord.com/channels/1136203602898194542/1155445615430471793/1181412895683248209), here's a quote from my reply:

> I'm not sure right now if this is a good decision. I'll revisit it once the Yazi plugin system is out.
> 
> I'm still exploring it and also checking what the plugin system can't accomplish and if there's a possibility for further expansion.

**Comment by jmlucjav** [comment]
thanks for the pointer! To be clear, I am not advocating for a multi process approach, if mc style operations are possible now, that would be enough for me.
